andela/AndelaEatsUI
# #163876985 Reports for the Admin UI

#### What does this PR do?
 It creates the UI for the Admin reports

#### Description of Task to be completed?
 As an admin user, when I log into Andela Eats and navigate to the admin dashboard, I should be able to see a Report tab that will have different reports on Andela Eats.

#### How should this be manually tested?
 - Clone the repo and checkout the `admin-report-tab-ui-163876985` branch
 - Run the app and go to the `Admin` dashboard.
 - CLick on the last icon in the side nav, `Reports`.
 - At this point, you should see the Admin Reports UI.
#### What are the relevant pivotal tracker stories?
- [#163876985](https://www.pivotaltracker.com/story/show/163876985)

#### Background Context
 None
#### Screenshots (If Applicable)
 None, still a WIP
